packages/ui-utils/src/types.ts

```diff
@@ -266,6 +284,14 @@ An optional object to be passed to the API endpoint.
   body?: object;
 };
 
+export interface URLFile {
+  name: string;
+  type: string | undefined;
+  url: string;
+}
```

Comment:
Can this be the same as MessageFile above?

Reply:
done!